WidgetExtension/ar.lproj/WidgetExtension.strings

```diff
@@ -0,0 +1,28 @@
+"3d6HSO" = "Enter username";
```

Comment:
Using IDs as keys looks strange, but I guess, we'd have to add another Crowdin-thing if we want to manage widget-localizations there?

Reply:
Alrighty :) I will add a short comment to the localization-docs later.